app/models/setting.rb

```diff
@@ -229,6 +229,18 @@ def self.model_name
     ActiveModel::Name.new(Setting)
   end
 
+  def filter_encrypted_attrs(filtered_changes)
+    # Check if the value should be masked for this instance
+    if encrypted?
```

Comment:
```suggestion
    filtered_changes = super
    if encrypted?
```
I would add a call to `super` to make sure the default functionality also works in this case.

Reply:
This is a new method that doesn't appear to exist anywhere else. I wonder how this works.

Reply:
It comes from the audited gem: https://github.com/collectiveidea/audited/blob/f935285c27151dff179ef274776f865302d40b8d/lib/audited/auditor.rb#L308

Not the best solution I could think of, but a working one :shrug: